### PR TITLE
MWPW-129575 - Authorable Tooltip

### PR DESCRIPTION
**Pull request by auniverseaway**
**Features**
* Ability to author tooltips
* Ability to optionally position tooltip (top, left, right, bottom)

**Usage**
Italicize the following text:

:tooltip: | Milo is awesome
:tooltip: | left | Milo is fast

**Samples**
[Word](https://adobe.sharepoint.com/:w:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7B3E41568F-4E3B-419C-BCD9-8A0FC670ACAF%7D&file=tooltip.docx) | [Preview](https://tooltip--milo--adobecom.hlx.page/drafts/cmillar/tooltip?martech=off)

**Resolves**
MWPW-129575

### CODE REVIEW

**Comment by auniverseaway** [comment]
@narcis-radu I think you're 100% right and I think we need to write a lot more JS to be much better about detecting screen edges and when the tooltip will be outside those bounds. I'm going to let people smarter than me tackle that. My only contribution here is where I think the code should live and what the AX is like.

**Comment by auniverseaway** [comment]
@Blainegunn at the very least I think we fix the z-index issue @narcis-radu raised.

**Review by Blainegunn — APPROVED** [review]
a picture of perfection!

**Review by seanchoi-dev — APPROVED** [review]
Looks good to me.

File: libs/styles/styles.css [hunk 1]
```diff
@@ -152,6 +152,11 @@ img {
   height: auto;
 }
 
+/* Tooltip */
+em:has(.icon-tooltip, .icon-info) {
```
Comment (rgclayton) [inline libs/styles/styles.css:156]: @Blainegunn This (has() selector) is going to be a problem for older Safari. We currently support back to Safari 14

https://milo.adobe.com/wiki/test-automation/test-automation-framework#browser--platform-support-matrix

Comment (Blainegunn) [reply libs/styles/styles.css:156]: i'm on it

**Review by rgclayton — APPROVED** [review]
Looks good! 
Getting the description updated before we merge and an email is sent out to the mailing list "new feature".

**Pull request merged**